packages/server/src/tests/banners/bannerTargetingTests.ts

```diff
@@ -0,0 +1,38 @@
+import { TargetingTest } from '../../lib/targetingTesting';
+import { BannerTargeting } from '@sdc/shared/types';
+
+const isFootballMatch = (targeting: BannerTargeting): boolean => {
```

Comment:
Hi Tom. I'm wondering if there might be some value if we put the exclusions into an dedicated Object, which we can then perform the tests against?

    const exclusions = {
        'football': ['tone/minutebyminute', 'tone/matchreports'],
        'fashion': [], 
        'tv-and-radio': [], 
        'travel': [],
    };

Reply:
thanks for the suggestion!
I've pushed up a change with a typescript-friendly version of your code